examples/train.py

```diff
@@ -110,9 +133,11 @@ def run_model_stablebaseline(flow_params,
 def setup_exps_rllib(flow_params,
                      n_cpus,
                      n_rollouts,
+                     flags,
```

Comment:
will this break replay scripts?  are there any other handles to setup_exps_rllib?